src/ConfigFactory.Avalonia/Builders/DropdownControlBuilder.cs

```diff
@@ -0,0 +1,43 @@
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Layout;
+using ConfigFactory.Core;
+using ConfigFactory.Core.Attributes;
+using System.Reflection;
+
+namespace ConfigFactory.Avalonia.Helpers;
+
+internal static class DropdownBuilder
```

Comment:
Control builders should be public and must implement `ControlBuilder<T> where T : ControlBuilder<T>`

```suggestion
public static class DropdownBuilder : ControlBuilder<DropdownBuilder>
```

(If this is not a traditional control builder it should be documented as such, and/or moved to another location)

Reply:
**Update:** looking through the code more I can see this isn't a typical control builder. Might be better in a `Helper` namespace, or just documented as an internal helper.

Reply:
if you don't mind I'll just add an additional explanation to that class and set it as `public`.

Reply:
That's fine, also no need to make it public if it's not a real control builder